common/recommender/Baseline.js

```diff
@@ -317,6 +318,24 @@ class Baseline {
 
     return entries;
   }
+
+  _dedupeSites(entries) {
```

Comment:
I was thinking leaving it blank so that if we ever change the dedupe key we can still use the generic method